app/templates/introduction.html

```diff
@@ -42,7 +42,7 @@ <h2 class="saturn col-10@l">{{legal_text}}</h2>
           <dl class="dl">
             <dt class="dl__title mercury">PERIOD</dt>
             <dd class="dl__data mars">
-              <span>{{ meta.survey.start_date|format_date }} to {{ meta.survey.end_date|format_date}}
+              <span>{{ meta.survey.start_date|format_date }}{% if meta.survey.end_date %} to {{ meta.survey.end_date|format_date}}{% endif %}
```

Comment:
Original it was planned to do it in the template rather then a filter, will check with @ajmaddaford